index.bs

```diff
@@ -2730,6 +2730,9 @@ To <dfn>parse attribution scopes for source</dfn> from a [=map=] |map| and a pos
 1. If |attributionScopeLimit| is null:
     1. If |result| is not [=set/is empty|empty=], return an error.
     1. Return |result|.
+
+Note: Empty attribution scopes are not allowed to prevent the selection of both buckets with and without scopes when |attributionScopeLimit| is set to 1, which would effectively result in two scopes.
```

Comment:
`bucket` seems to be a novel term here, and is easy to confuse with the same term from aggregatable attribution. Do we mean `sources` instead?

Reply:
Good point. Done.